content/projects/gsoc/2023/project-ideas/displaying_plugin_health_scores.adoc

```diff
@@ -0,0 +1,44 @@
+---
+layout: gsocprojectidea
+title: "Displaying of Plugin Health Scores"
+goal: "Display a possibly filtered view of Plugin Health Scores on plugins.jenkins.io and the details for each individual plugin"
+category: Tools
+year: 2023
+status: discussion
+sig: platform
+skills:
+- Java
+- YAML
+- Command line tools
+- Package management tool theory
+mentors:
+- "jleon"
+links:
+    emailThread: https://community.jenkins.io/t/gsoc2023-project-idea-displaying-of-plugin-health-scores/5318
+//   gitter: "jenkinsci/plugin-installation-manager-cli-tool"
+//   draft: https://docs.google.com/document/d/1s-dLUfU1OK-88bCj-GKaNuFfJQlQNLTWtacKkVMVmHc
```

Comment:
```suggestion
//   gitter: "TBD"
//   draft: TBD
```

Hi @jmMeessen

Thank you for the clarification.

To improve context, @jmMeessen wrote, this "is a placeholder draft description in the commented out section. It points to an outdated and different project idea."
(https://github.com/jenkins-infra/jenkins.io/pull/5926/files#r1071309830)

@jmMeessen While these are comments, can the outdated links be removed or changed to "TBD" (per convention below)?

Without improved context, I had clicked through the links (even though they are comments), and I did not fully realize that they refer to an outdated and different project idea.

My mistake was having missed reading the `//` in both the `gitter` and `draft` lines.

In addition to the Google Docs link, I even tried searching for gitter channel, and I had trouble finding that.

My suggestion is to change these two lines to use "TBD" values to avoid the confusion and mistake that I had, because I did not have the full context.

Note: I suggest "TBD" because that's the convention used in following lines / immediately below.